pandas/core/groupby.py

```diff
@@ -1055,6 +1056,59 @@ def tail(self, n=5):
         tail = obj[in_tail]
         return tail
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Apply func(self, \*args, \*\*kwargs)
+
+        .. versionadded:: 0.17.0
+
+        Parameters
+        ----------
+        func : function
+        function to apply
```

Comment:
Can you format this section following the numpydoc standard? (see https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections)